marimo/_cli/sandbox.py

```diff
@@ -156,6 +156,11 @@ def prompt_run_in_sandbox(name: str | None) -> bool:
         if GLOBAL_SETTINGS.YES:
             return True
 
+        # Check if not in an interactive terminal (i.e. Docker)
+        # default to False
+        if not sys.stdin.isatty():
+            return False
```

Comment:
i think `False` is better for the case for Docker, but not sure about others

Reply:
maybe true for sandbox?

Reply:
I think `False` is the better default for sandbox IN when running Docker (which is not an interactive termainl)

Reply:
otherwise you pay installation time on cold-starts which is a bad pattern, and doing `True` would just hide that